extension/chrome/settings/inbox/inbox.ts

```diff
@@ -132,6 +154,29 @@ export class InboxView extends View {
     Xss.sanitizeRender('h1', title);
   };
 
+  private preRenderSecureComposeInFullScreen = () => {
+    if (this.useFullScreenSecureCompose && this.thunderbirdMsgId) {
+      this.injector.openComposeWin(undefined, true, this.thunderbirdMsgId, this.composeMethod);
+    }
+  };
+
+  private parseThreadIdFromHeaderMessageId = async () => {
+    let threadId;
+    const tbMsgId = Number(this.thunderbirdMsgId);
+    if (Catch.isThunderbirdMail() && tbMsgId && !this.useFullScreenSecureCompose) {
```

Comment:
no need to convert `this.thunderbirdMsgId` to `number`, as it's already has `number` type. we can just check if it's defined in `if`:
```ts
if (Catch.isThunderbirdMail() && this.thunderbirdMsgId && !this.useFullScreenSecureCompose) {
```

Reply:
Okay - will update. thanks!